### DISCUSSION TITLE
Question about transfering repositories

### DISCUSSION THREAD

**Discussion by RRicardo-Lima** [community/community#67653] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hello,

I hope you're all doing well.

I wanted to discuss my GitHub accounts situation. At present, I have two separate accounts, and my primary objective is to centralize all my work under a single account. Eventually, I plan to remove the extra account. While I believe that transferring the repositories should help me achieve this, I want to proceed with caution to ensure I don't lose any of my previous work.

Could someone kindly provide me with a clear, step-by-step guide on how to execute this process?

Additionally, I'm interested in understanding what will happen to my contributions. After the repository transfer, is it possible to have all my contributions consolidated and displayed in a single account?

**Comment by pedromelobitencourt** [comment, marked as answer]
Certainly, I'll provide you with simplified answers to your questions:

How can I move repositories from one GitHub account to another?
* Log in to the source account.
* Navigate to the repository.
* Click on the "Settings" (gear icon).
* Select "Transfer" and follow the instructions to transfer it to the destination account.

What happens to my contributions after transferring repositories?
* New contributions will be associated with the destination account.
* Old contributions will remain with the source account, and there's no automated way to transfer them.

Can I consolidate all contributions in one account after transferring repositories?
* No, historical contributions will still be linked to the original account.

What about my GitHub accounts following the transfer?
* You can choose to retain both accounts or archive/delete the source account.
* Keep in mind that transferring repositories won't move your historical contributions. New contributions will be attributed to the destination account, but you can't merge old contributions from the source account with the new one.
